packages/kg-lexical-html-renderer/lib/transformers/element/image.js

```diff
@@ -0,0 +1,17 @@
+const {$isImageNode} = require('../../nodes/ImageNode');
+
+module.exports = {
+    export(node) {
+        if (!$isImageNode(node)) {
+            return null;
+        }
+        return (`
+        <figure>
+        <img src="${node.src}" alt="${node.altText}" /> 
+        <figcaption>
+            ${node.caption}
+        </figcaption>
+        </figure>
```

Comment:
Can we try to keep indentation with any html like this? Makes it easier to follow, especially as the output gets more complex.